### ISSUE TITLE
Improving Abstract Review Efficiency

### ISSUE THREAD

**Issue by Bmcoimbra** [asreview/asreview#1688]
Dear colleagues,

I found myself pondering the other day as I delve into a systematic review of prospective studies on stress disorders. While ASReview undoubtedly streamlines the entire process, I still find myself sifting through numerous abstracts of studies with ineligible methodological designs. Is it perhaps feasible to incorporate features that allow for the highlighting of specific words in the abstracts to expedite the process? For instance, if I'm conducting a review of prospective studies, the ability to highlight terms like "cross-sectional" and "retrospective" would greatly aid in swiftly discerning and discarding irrelevant papers. Keep up the great work!

**Comment by jpripamonti** [comment]
I agree with this. Having the capacity to highlight different terms in the abstracts would make the reviewing more efficient.
